Jenkinsfile

```diff
@@ -135,18 +133,14 @@ node('java') {
                     // Create tag.
                     def tag = buildConfig.tagName + "-" + buildConfig.releaseVersion
                     sh "git tag $tag"
-                    wrap([$class: 'MaskPasswordsBuildWrapper', varPasswordPairs: [[password: 'GITHUB_API_KEY', var: 'SECRET']]]) {
```

Comment:
I suspect that removing will cause the password to be included in the build log.
Let me know if you tested this and the key is masked in the log.